### PR TITLE
Add IE11 support instructions to readme

### PR DESCRIPTION
**Pull request by maxmumford**
Hiya,

I spent some time trying to debug why this lib wasn't working in IE11 before I spotted that a polyfill is required to make the angular-inviewport library work, so I've added it to the readme.

Cheers!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,30 @@ a.active {
 }
 ```
 
+## Browser Support
+
+This library makes use of the [Intersection Observer API][intersection-observer-api] which requires a [polyfill][intersection-observer-polyfill] to work on some browsers.
```
Comment (edoparearyee) [inline README.md]: Just missing some links here.

```suggestion
This library makes use of the [Intersection Observer API](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API) which requires a [polyfill](https://github.com/w3c/IntersectionObserver/tree/master/polyfill) to work on some browsers.
```

**Review by edoparearyee — CHANGES_REQUESTED** [review]
@maxmumford thanks for updating this, just missing some links in the markdown otherwise this is great!

**Comment by maxmumford** [comment]
Oops my bad - just copied and pasted from the other project's readme but didn't check the links after doing so. Should be updated now? Cheers

**Comment by maxmumford** [comment]
Oops my bad - just copied and pasted from the other project's readme but didn't check the links after doing so. Should be updated now? Cheers

**Review by edoparearyee — APPROVED** [review]

**Pull request merged**